docs/content/community/presentations/index.md

```diff
@@ -0,0 +1,25 @@
+---
+type: docs
+title: "Give a Radius presentation"
+linkTitle: "Presentations"
+description: "Learn how to give a Radius presentation, with downloadable slides and assets"
+weight: 400
+---
+
+We welcome community members giving presentations on Radius and spreading the word about all the awesome Radius features!
+
+## Giving a Radius presentation
+
+1. Begin by downloading the Radius Presentation Deck. This contains slides and diagrams needed to give a Radius presentation.
```

Comment:
Shoiuld we add a hyperlink to https://calm-bush-0cc2a4f1e-1020.westus2.3.azurestaticapps.net/community/presentations/#slides for `Radius Presentation Deck`?